modules/builtin/src/content-types/text.js

```diff
@@ -130,31 +130,9 @@ module.exports = {
     ],
     fields: [
       {
-        type: 'text',
+        type: 'custom',
         key: 'text',
-        label: 'module.builtin.types.text.message'
```

Comment:
We already have some logic that we use everywhere for custom components and I think it should also be used for this. When you have a type custom, you can provide the name of the module, the name of the component, and provide additional properties.

This way, we have a standardized way and this lets module developers provide additional features.

![image](https://user-images.githubusercontent.com/42552874/85177966-4483c880-b24b-11ea-9fa2-ecb09b5869d3.png)


You can use this snippet of code in the form: 
```js
<InjectedModuleView moduleName={module} componentName={component} lite extraProps={props} />
```

Reply:
Think there might've been a misunderstanding of what I was trying to achieve. After talking with @EFF for a while here's what the conclusion was. The word custom was misleading, I replaced it for override as this type is only meant to be used internally to override some components when the data structure or UI is slightly different than an existing component in the shared library. In this instance, the data for the content type `text` was ```{text: 'some text', variations: ['some variation', 'some other variation]}``` but was expected to behave like ```{ variations: ['some text', 'some variation', 'some other variation]}``` so it was easier to wrap it in a different component to manipulate the data than adding many conditions. 

The reason why the text_array type doesn't exist yet on the form is that there's no other occurrence using it yet, but it will be done in the next PR I'm currently working on